.github/workflows/buildtest.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - name: Go test
         if: ${{ matrix.goarch }} == "amd64"
-        run: sudo go test ./... # sudo needed for netns change in test
+        run: sudo go test -race ./... # sudo needed for netns change in test
```

Comment:
I also saw there is a an option in the Makefile to run the tests with the race flag ( make test-race). 
I wasn't sure why the CI doesn't use the Makefile to run the tests - so i decided to do minimal changes. But if there isn't any reason not to use the option in the Makefile i think its better to use it.

Reply:
Test rules in Makefile need cleaning, but they can be done in a different PR.